modules/violations-page-overview.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * operating/respond-to-violations.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="violations-page-overview_{context}"]
+= Violations page overview
+
+The *Violations* page shows a list of violations and organizes information into the following groups:
+
+* *Policy*: The name of the violated policy.
+* *Entity*: The entity where the violation occurred.
+* *Type*: The type of entity.
++
+The following values are associated with the type of entity: 
++
+** `deployment`
+** `namespace`
+** `cluster`
```

Comment:
The type includes more options like `Pod` and `DaemonSet`. I don't know if we wanted to include the full list. Charmik can you share the list of those options here?

Reply:
For workload violations the type will show workload type like Deployment, Pod, DaemonSet, CronJob, StatefulSet, ReplicaSet, StaticPods etc. 
For other resources violations, the type will be the resource type: Secrets, Configmaps, Cluster_roles, Cluster_role_bindings, Network_policies, Security_context_constraints,  Egress_firewalls

Also, It would not be namespace or cluster